### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Janiceilene**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation
  

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,48 +1,11 @@
...
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  
-Each active repo has its own channel to direct questions to (for example #powerup or #portal).  Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](https://github.com/systers/systers.github.io/blob/master/code_of_conduct.md).
-
-[General Guidelines](#general-guidelines)  
-[Opening a New Issue](#opening-a-new-issue)  
-[Important Reminders](#important-reminders)  
-[Communicate](#communicate)  
-[Don’t Know Code?](#dont-know-code)  
+* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](systers.io/code-of-conduct.html).
```
Comment (sara-02) [inline CONTRIBUTING.md]: @Janiceilene just a suggestion, instead of `this is an..` we can have `Systers is an `

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,48 +1,11 @@
...
 ## General Guidelines
+* If you’re just getting started work on an issue labeled “First Timers Only” in any project. additional resources are available on our [website](systers.io).
```
Comment (sara-02) [inline CONTRIBUTING.md]: Minor nitpick `getting started, pick issues labeled...`.
Capital `Additional`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,48 +1,11 @@
...
 * In an active repository (not an archived one), choose an open issue from an from the issue list, claim it in the comments, and a maintainer will assign it to you.
```
Comment (sara-02) [inline CONTRIBUTING.md]: Looks like we missed a double `from`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,48 +1,11 @@
...
-* After approval you must make continuous notes on your progress in the issue while working.  If there is not at least one comment every 3 days, the maintainer can reassign the issue.  
-* If you have any questions, you can post them as a comment in your issue or in the project specific slack channel.
-* Once you’ve fixed the issue, send us a pull request.
-* Send all pull requests to the correct branch:  
-  * Send Code pull requests to the develop branch.
-  * Send Quality Assurance pull requests to the testing branch.
-  * Send User Interface, Outreach, and Documentation pull requests to the non-coding branch.
-## Opening a New Issue
+* After approval you must make continuous notes on your progress in the issue while working.  If there is not at least one comment every 3 days, the maintainer can reassign the issue.
+* When sending a PR have an appropriate title referencing the issue which it solves. Add “fixes #<issue-number>” in the commit body, so that when the PR gets merged, the issue get closed automatically. Do not do this if the PR solves only a part of the issue. See more information on commit guidelines [here](https://udacity.github.io/git-styleguide/).
```
Comment (sara-02) [inline CONTRIBUTING.md]: `issue gets closed` or `issue is closed`

File: CONTRIBUTING.md (hunk 3 above)
Comment (Janiceilene) [reply CONTRIBUTING.md]: Thanks!

**Comment by sara-02** [comment]
@Janiceilene Thanks, LGTM

**Comment by MeepyMay** [comment]
Done! Thanks.

**Pull request merged**